kiali.yaml

```diff
@@ -0,0 +1,75 @@
+package:
+  name: kiali
+  version: 1.87.0
+  epoch: 0
+  description: The Console for Istio Service Mesh
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - kiali-api
+      - kiali-ui
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+      - nodejs
+      - npm
+      - tree
+      - yarn
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kiali/kiali
+      tag: v${{package.version}}
+      expected-commit: 1913b4cfd5d0f1d4b2261b990fead586c6eed876
+
+subpackages:
+  - name: ${{package.name}}-api
+    description: Kiali API
+    dependencies:
+      runtime:
+        - ca-certificates
+    pipeline:
+      - uses: go/build # Builds to /opt/kiali/kiali
+        with:
+          packages: .
+          prefix: opt
+          install-dir: kiali
+          output: kiali
+          ldflags: |
+            -X main.version=v${{package.version}} \
+              -X main.commitHash=$(git rev-parse HEAD) \
+              -X main.goVersion=$(go version | grep -Eo '[0-9]+\.[0-9]+\.[0-9]+')
+      - uses: strip
```

Comment:
just a quick note, we don't need to use strip here since go/build pipeline already takes care of stripping the binaries via the -w flag of go build command.